build/config/ios/ios_sim.py

```diff
@@ -4,11 +4,16 @@
 # found in the LICENSE file.
 
 import argparse
-import os
 import errno
+import os
+import re
 import subprocess
 import sys
-import re
+import time
+
+IOS_SIM_PATH = [
+  '/Applications/iOS Simulator.app/Contents/MacOS/iOS Simulator'
```

Comment:
`open -a iOS\ Simulator` and similar variants don't work for me.  Does it work for you?